ui/components/component-library/button-primary/button-primary.scss

```
@@ -0,0 +1,44 @@
+.mm-button-primary {
+  color: var(--color-primary-inverse);
+  background-color: var(--color-primary-default);
+
+  &:hover {
+    color: var(--color-primary-inverse);
+    box-shadow: var(--component-button-primary-shadow);
+  }
+
+  &:active {
+    color: var(--color-primary-inverse);
+    background-color: var(--color-primary-alternative);
+  }
+
+  &--type-danger {
+    color: var(--color-error-inverse);
+    background-color: var(--color-error-default);
+
+    &:hover {
+      color: var(--color-error-inverse);
+      box-shadow: var(--component-button-danger-shadow);
+    }
+
+    &:active {
+      color: var(--color-error-inverse);
+      background-color: var(--color-error-alternative);
+    }
+  }
+}
+
+
+.mm-button-primary.mm-button--disabled {
```

Nit: Does simplifying this work? Then if the root className is changed don't have to change it here. Will need to fix formatting and add closing bracket

```suggestion
  &.mm-button--disabled {
```